clients/go/marshal.go

```diff
@@ -0,0 +1,37 @@
+package raccoon
+
+import (
+	"errors"
+
+	"encoding/json"
+
+	"google.golang.org/protobuf/proto"
+)
+
+type Message interface{}
+
+var (
+	// json clickstream marshaler
```

Comment:
@AkbaraliShaikh Let's use raccoon as a ref instead of clickstream.

Reply:
updated